Lib/test/test_logging.py

```diff
@@ -3932,15 +3950,21 @@ def test_config_queue_handler(self):
     @support.requires_subprocess()
     @patch("multiprocessing.Manager")
     def test_config_queue_handler_does_not_create_multiprocessing_manager(self, manager):
-        # gh-120868
+        # gh-120868, gh-121723
 
         from multiprocessing import Queue as MQ
 
         q1 = {"()": "queue.Queue", "maxsize": -1}
         q2 = MQ()
         q3 = queue.Queue()
-
-        for qspec in (q1, q2, q3):
+        # FuzzyCustomQueueProtocol pass the checks but will not be usable
+        # since the signatures are incompatible. Checking the Queue API
+        # without testing the type of the actual queue is a trade-off
+        # between usability and the work we need to do in order to safely
+        # check that the queue object is a valid queue object.
```

Comment:
```suggestion
        # FuzzyCustomQueueProtocol passes the checks but will not be usable
        # since the signatures are incompatible. Checking the Queue API
        # without testing the type of the actual queue is a trade-off
        # between usability and the work we need to do in order to safely
        # check that the queue object correctly implements the API.
```